app/models/alert_type.rb

```diff
@@ -46,6 +46,14 @@ def cleaned_template_variables
     end
   end
 
+  def available_charts
+    constant_class = class_name.constantize
```

Comment:
This is obviously yuck, but will get better when fixed in analytics. We get the template variables, create an analytics alert, then send it the variable names to get the chart config names

Reply:
I think we need to revisit how the charts are referenced in the analytics. I think the `variable name` -> `chart name` translation should be available on the class of the alert and done when we want to save the alert or render the chart. If we do the translation at the point we display and save the content then it's effectively frozen at that point and any change to the analytics (e.g. `one_year_baseload_chart` references `:alert_1_year_baseload_with_carbon` instead of `:alert_1_year_baseload`) won't be reflected until we revisit the content again and `:alert_1_year_baseload` will no longer be an option in the list to select.

It feels like this way round we're saving the value of one of the variables in the content, not the name of the variable.

Reply:
Actually, we get the variable names in chart data now anyway, so I think we can just persist the variable name here and cross reference it when we pull the chart out.

Reply:
So, ignore the bit about making it available on the class